bitnami/spark/README.md

```diff
@@ -114,6 +115,8 @@ The following tables lists the configurable parameters of the spark chart and th
 | `worker.readinessProbe.timeoutSeconds`      | When the probe times out                                                                           | 5                                                       |
 | `worker.readinessProbe.failureThreshold`    | Minimum consecutive failures for the probe to be considered failed after having succeeded.         | 6                                                       |
 | `worker.readinessProbe.successThreshold`    | Minimum consecutive successes for the probe to be considered successful after having failed        | 1                                                       |
+| `worker.podAnnotations` | Additional annotations for worker metrics exporter                                                          | `{prometheus.io/scrape: "true", prometheus.io/port: "8081" prometheus.io/path: "/metrics/prometheus}` |
```

Comment:
I would make the description just "Additional annotations for the worker" since this is not exclusive for metrics. The default value is also `{}`.

Reply:
fixed